test/specs/commands/create_transaction_cast_votes.js

```diff
@@ -74,8 +74,8 @@ describe('create transaction cast votes', () => {
 																			then.itShouldNotGetTheDataUsingTheUnvotesSource,
```

Comment:
Do we not have any tests for transactions with votes and unvotes at the same time?

Reply:
It should.
https://github.com/LiskHQ/lisky/pull/471/commits/0a3a7ee1ce64db0388b143a3a6a9676492973a37#diff-3b2b6706fb38e00a91285dd3f43c8774L37

this one is the regarding test I think.

Reply:
Right, but the name of that step should be changed now we're not prepending anything, no?

Reply:
yes that is true! Updated!